### ISSUE TITLE
Find text styles broken and give a text style

### ISSUE THREAD

**Issue by ernestoybk** [rojcyk/figma-reconnect#9]
**This is necessary**
I think this feature with the corret marketing is all figma user need. There was another plugin that do it, but it didnt work correctly. So maybe you could do it. I and a lot of user more are ready to pay for it. 
**The plugin that don´t work correctly and die**
https://www.figma.com/community/plugin/751892393146479981/Roller-%C2%B7-Design-Linter

**Comment by rojcyk** [comment]
@ernestoybk could you elaborate a bit more? Describe how would you like it to work.

**Comment by ernestoybk** [comment]
Of course, It import your text and color styles and then you can scan all the page and find and count the ones that don't have style as errors. Then You can open the error and asign a text or color style, in one of the words or all.

**Comment by ernestoybk** [comment]
> @ernestoybk could you elaborate a bit more? Describe how would you like it to work. @rojcyk 

Of course, It import your text and color styles and then you can scan all the page and find and count the ones that don't have style as errors. Then You can open the error and asign a text or color style, in one of the words or all.

**Comment by rojcyk** [comment]
@ernestoybk  It should already work like this. The only difference is that you can not make it for a whole document as that could take a **very** long time but only for your selection.
